MotorControle/Platform_IO_Board_Motor_Controle/test/test_main.cpp

```diff
@@ -17,11 +18,12 @@ void loop()
     //initialisation
     TestComm test_comm;
     Test_Join test_join;
+    Test_DriveManager test_drivemanager;
 
     //lunch_ de_test
```

Comment:
launch the test

Reply:
lol hahah, la faute vient pas de moi pour celle-là #dyslexie